### PR TITLE
Added support for (non-numeric) information fields for each operator

### PR DESCRIPTION
**Pull request by bstocks101**
Modified the operator data flow to allow for generic, non-numeric values to be stored in the data structure and reported remotely as a new entry for an operator. This new entry will register as a new metric and display its value in the "info_string" field.

### CODE REVIEW

File: benchmarking/frameworks/caffe2/caffe2.py [hunk 1]
```diff
@@ -342,7 +342,10 @@ def _processDelayData(self, data):
             for k, v in d.items():
                 for kk, vv in v.items():
                     key = k + " " + kk
-                    details[key]["values"].append(float(vv["value"]))
+                    if "info_string" in vv:
+                        details[key]["info_string"] = vv["info_string"]
```
Comment (sf-wind) [inline benchmarking/frameworks/caffe2/caffe2.py]: Here there are multiple values, may we assert if the values don't match?

Comment (bstocks101) [reply benchmarking/frameworks/caffe2/caffe2.py]: I'm sorry, I'm not sure I understand. What multiple values are you concerned with?

Comment (bstocks101) [reply benchmarking/frameworks/caffe2/caffe2.py]: Sorry just following up on this comment. Can you please clarify which multiple values you are referring to and which assertions should be made?

File: benchmarking/frameworks/caffe2/caffe2.py [hunk 2]
```diff
@@ -342,7 +342,15 @@ def _processDelayData(self, data):
             for k, v in d.items():
                 for kk, vv in v.items():
                     key = k + " " + kk
-                    details[key]["values"].append(float(vv["value"]))
+                    if "info_string" in vv:
+                        if "info_string" in details[key]:
+                            assert details[key]["info_string"] == vv["info_string"], \
+                                "info_string values for {} ".format(key) + \
```
Comment (sf-wind) [inline benchmarking/frameworks/caffe2/caffe2.py]: better print out the two non-matching info strings

**Pull request merged**